scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -107,5 +108,28 @@ Use the *rhel-tools* container with the following:
 
 See the link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-11-using-the-atomic-tools-container-image[RHEL Tools Container documentation] for more information.
 
-
-
+[[scaling-performance-debugging-using-openshift-ansible]]
+== Debugging Using openshift-ansible Health Checks
+
+Diagnostic health checks are available through *openshift-ansible*,
+the Ansible-based tooling used to install and manage {product-title} clusters.
+They can report common deployment problems for the current {product-title}
+installation.
+
+These checks can be run either using the `ansible-playbook` command (the same
+method used during
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation]) or using the Docker CLI to directly run a
+link:https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[containerized version] of *openshift-ansible*. For the `ansible-playbook` method, the checks
+are provided with the *atomic-openshift-utils* RPM package. For the Docker CLI
+method,
+ifdef::openshift-enterprise[]
+the *openshift3/ose-ansible* container image is distributed via the
+link:https://registry.access.redhat.com[Red Hat Container Registry].
+endif::[]
+ifdef::openshift-origin[]
+the *openshift/origin-ansible* container image is distirbuted via Docker Hub.
```

Comment:
Is the content duplication intentional? This paragraph is the same in two files (typo included).